### PR TITLE
Sentinel's Compromise tweaks

### LINKED ISSUE #6301: Free cogscarab heals
## Round ID:
35864

## Testmerges:
[#5755: 'Updates Kilo Station and re-adds it to rotation' by dog132-ss13 at commit 91a43e0517](https://www.github.com/beestation/beestation-hornet/pull/5755)
[#6014: 'get_hear and get_hearers_in_view optimisations' by PowerfulBacon at commit 556c5084ed](https://www.github.com/beestation/beestation-hornet/pull/6014)

## Reproduction:
1. Be a cogscarab during the Ark fight
2. Cast Sentinel's Compromise, which allows you to heal an ally within sight but makes you take half the damage healed as toxin damage
3. Use it on an ally, fully prepared to die
4. Ally is healed and you have taken no damage because you are a simple mob and can't take toxin damage

### PR DESCRIPTION
**Pull request by Penwin0**
## About The Pull Request
Compromise now checks if the invoker is /mob/living/carbon/human, namely to prevent cogscarabs from abusing it with no downside. closes #6301
Oozelings now also take damage from Sentinel's compromise. 

## Why It's Good For The Game
Cogscarabs should not be able to make someone nigh unkillable with no downside.
Oozelings should not be able to bypass this either. 

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:
balance: Scarabs can't use Sentinels compromise anymore.
tweak:Oozelings now take toxin damage from Sentinel's compromise.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
Pretty sure slime people will be able to heal from it too since the toxin damage isn't forced.
Also you should give a message in chat as to why its not working rather than just have it not work.

**Comment by Penwin0** [comment]
There you go ol chap

File: code/modules/antagonists/clock_cult/scriptures/sentinels_compromise.dm [hunk 1]
```diff
@@ -47,5 +50,5 @@
 	new /obj/effect/temp_visual/heal(get_turf(M), "#f8d984")
 	playsound(M, 'sound/magic/magic_missile.ogg', 50, TRUE)
 	playsound(invoker, 'sound/magic/magic_missile.ogg', 50, TRUE)
-	invoker.adjustToxLoss(min(total_damage/2, 80))
+	invoker.adjustToxLoss(min(total_damage/2, 80), FALSE, TRUE)
```
Comment (PowerfulBacon) [inline code/modules/antagonists/clock_cult/scriptures/sentinels_compromise.dm]: This should update health, as its the last adjust called. 
```suggestion
	invoker.adjustToxLoss(min(total_damage/2, 80), TRUE, TRUE)
```

**Review by PowerfulBacon — APPROVED** [review]

**Comment by Isy232** [comment]
holy fuck yes thanks so much this was bullshit

**Pull request merged**